sdcm/sct_config.py

```diff
@@ -1785,6 +1789,10 @@ def __init__(self):
                     raise ValueError(
                         f" Got error: {repr(exp)}, on item '{param}'") from exp
 
+        # 15 Force endpoint_snitch to GossipingPropertyFileSnitch if using simulated_racks
+        if self.get("simulated_racks") > 1:
+            self["endpoint_snitch"] = "org.apache.cassandra.locator.GossipingPropertyFileSnitch"
```

Comment:
This `silent` overwriting looks dangerous to me.
Maybe we should check the value of this option and if it is already set then fail the test.
If it is not set then define it like here.

Reply:
I think failing on the configuration level in this case seems o.k. to me, and should be enough

Reply:
ok